client/src/components/Project/Scrums/index.jsx

```diff
@@ -0,0 +1,264 @@
+              assignee:'',
+              ticket:''
+          }],
+        })
+      )
+    };
+
+    handleAddTask = (event, column) => {
+      const newTask = event.target.value;
+      this.setState(prevState => {
+        const clonedTasks = [...tasks];
+        clonedTasks[clonedTasks.length - 1][column] = newTask;
+        return { tasks: clonedTasks, newRow: clonedTasks[clonedTasks.length - 1] };
+      });
+    };
+
+    handleEditTask = (event, record, column) => {
+      const { tasks } = this.state;
+
+      //Add a new task
+      if (record.id === tasks[tasks.length - 1].id) {
+        this.setState({ saving: true, saved: false })
+        return this.handleAddTask(event, column);
+      }
+
+      //Edit Task
+      const newTask = event.target.value;
+      const taskId = record.id;
+      const clonedTasks = [...tasks];
+      const updateTask = clonedTasks.find(task => task.id === taskId);
+      updateTask[column] = newTask;
+
+      //Validate
+      if (this.validateTask(updateTask)) {
+        this.updateTasks(updateTask, clonedTasks);
+      }
+    };
+
+    updateTasks = (task, tasks) => {
+      this.setState({ tasks, saved: true });
+    };
+
+    validateTask = (task) => {
+      this.setState({ taskDescriptionErr: false });
+      if (task['task_description'].length < 1) {
+        this.setState({ taskDescriptionErr: true });
+        return false;
+      }
+    }
+
+    handleDeleteTask = (event) => {
+      const { tasks, rowSelected } = this.state;
+      const deletedTask = tasks.filter(task => task.id === rowSelected)[0];
+      this.deleteSwal(deletedTask);
+    }
+
+    deleteSwal = () => {
+      return Swal.fire({
+        type: 'warning',
+        text: 'Are you sure to delete this task?',
+        showConfirmButton: true,
+        showCancelButton: true,
+        className:'deletTaskSwal'
+      })
+    };
+
+    showSwal = deletedTask => {
+      this.deleteSwal().then(response => {
+        if (response.value) this.confirmDelete(deletedTask);
+      });
+    };
+
+    confirmDelete = deletedTask => {
+      //Fetch 
+    };
+
+      columns = [
+          {
+            title: 'Task',
+            dataIndex: 'task_description',
+            render: (value, record) => {
+              return (
+                <Editable
+                  html={value}
+                  onChange={event => this.handleEditTask(event, record, 'task_description')}
+                  tagName="span"
+                  className="tasks__cell description"
+                />
+              );
+            },
+          },
+          {
+            title: 'Action Type',
+            dataIndex: 'action_type',
+            render: (value, record) => {
+              return (
+                <ActionTypeSelect onChange={event => this.handleEditTask(event, record, 'action_type')} defaultValue={record.action_type} />
+              );
+            },
+            filters: Filter(this.state.tasks).actionsFilters,
```

Comment:
the filters won't be updated here because when the state changes this code won't be called
so move this line inside the render function
```js
this.columns[1] = Filter(this.state.tasks).actionsFilters;  // you can also destructure tasks from this.state
```